package.json

```diff
@@ -68,6 +70,7 @@
     "bundlesize": "0.17.0",
     "chai": "4.2.0",
     "chai-as-promised": "7.1.1",
+    "chromedriver": "2.45.0",
```

Comment:
Curious: How large is this dependency? I don't suppose it can be lazy-installed it like you did with `@percy/puppeteer` in #19341, given that the tests will need to run on Travis and during local development? (I think the answer is no, but figured it's worth a mention.)

Reply:
With `selenium-webdriver`, totals to 26M. Tracking it here #20320 and will update this once I create the gulp task

Reply:
Aha! IIRC, `puppeteer` was 300M in size, and was slowing down the default developer workflow even though most people don't run the visual tests locally. At 26M, I'm not so concerned.

Reply:
It'll increase once we add new browsers, so still worth doing :)